src/Form/Form.js

```diff
@@ -49,22 +48,24 @@ class Form {
         }
 
         this.getValues = () => {
-            const credentials = [...this.inputs.credentials, ...this.inputs.emailNew].reduce((output, input) => {
+            const credentials = [...this.inputs.credentials, ...this.inputs.identities].reduce((output, input) => {
```

Comment:
When I moved the email to identities I forgot to cleanup a lot of this logic. Here I am doing just that. (I've also created regression testing below)